packages/retail-ui-extensions/src/extension-api/scanner-api/scanner-api.ts

```diff
@@ -0,0 +1,24 @@
+import type {RemoteSubscribable} from '@remote-ui/async-subscription';
+
+/** The scanner source the POS device supports */
```

Comment:
<img width="474" alt="Screenshot 2023-04-20 at 9 55 08 AM" src="https://user-images.githubusercontent.com/46533923/233388465-a207606d-0bb8-4a76-82e0-a223792a82ce.png">

More or less for the inline docs. I know the code is pretty self documenting, but it's always good to make sure everyone knows what's going on in case it's not clear 😄